kasa/smartcam/modules/camera.py

```diff
@@ -15,6 +16,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 LOCAL_STREAMING_PORT = 554
+ONVIF_PORT = 2020
+
+
+class StreamResolution(StrEnum):
```

Comment:
```suggestion
class StreamType(StrEnum):
```
Maybe? I'm wondering if we should simply rename `stream_rtsp_url` to  `stream_url` and add a a type for ONVIF. What do you think?

Reply:
Fair enough. I'm wondering if there are other stream resolutions besides HD and SD available, basically asking myself, whether we should just return a dict of all available streams through a single property?

Reply:
Okay, sure! I think this is fine as it is and we can revisit it later when really needed.